data/pilots/scum-and-villainy/firespray-class-patrol-craft.json

```diff
@@ -195,6 +195,22 @@
       "standard": true,
       "extended": true,
       "epic": true
+    },
+    {
+      "name": "Hondo Ohnaka",
```

Comment:
noted in slack but hondo's card says "Firespray-class Patrol **Ship**", not "Craft". I don't know how this could ever matter, but mentioning just in case I'm missing something.

Reply:
That may or may not change on the printed card anyway